packages/fields/types/Content/views/src/Field.js

```diff
@@ -1,57 +1,13 @@
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
-import { useState, useMemo } from 'react';
+import { useState } from 'react';
 import Editor from './editor';
 import { Value } from 'slate';
 import { initialValue } from './editor/constants';
-import * as paragraph from './editor/blocks/paragraph';
 import { FieldContainer, FieldLabel, FieldInput } from '@arch-ui/fields';
 import { inputStyles } from '@arch-ui/input';
 
 let ContentField = ({ field, value: serverValue, onChange, autoFocus }) => {
-  const blocksModules = field.adminMeta.readViews(field.views.blocks);
-  let blocks = useMemo(() => {
-    let defaultBlocks = [paragraph];
-
-    let customBlocks = blocksModules.map((block, i) => ({
-      ...block,
-      options: field.config.blockOptions[i],
-    }));
-
-    let combinedBlocks = [...defaultBlocks, ...customBlocks];
-
-    let flatBlocks = [];
-
-    function pushBlocks(blocksToPush) {
-      blocksToPush.forEach(block => {
-        let existingItem = flatBlocks.find(({ type }) => type === block.type);
-        if (existingItem === undefined) {
-          let { dependencies, ...blockToInsert } = block;
-          flatBlocks.push(blockToInsert);
-          if (dependencies !== undefined) {
-            pushBlocks(dependencies);
-          }
-        } else if (existingItem.Node === undefined) {
```

Comment:
This condition would previously only ever be hit when a block is pushed _more than once_. Which made it possible for a block without the required `.Node` property to be pushed as long as it wasn't pushed twice. I have fixed this in my refactor.